zombienet_tests/misc/0001-check_paritydb.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+
+const { exec } = require("child_process");
+
+// Check if we are running paritydb on a specified node.
+async function run(nodeName, networkInfo, jsArgs) {
+    console.log(networkInfo);
+    let node_path = networkInfo.tmpDir;
+    // TODO: We need Zombienet to provide the chain spec name in networkInfo to un-hardcode this path.
+    let dir = `/data/chains/rococo_local_testnet/paritydb/full`;
+
+    const { exec } = require("child_process");
+
+    exec(`ls -lahR /data`, (error, stdout, stderr) => {
```

Comment:
Hi @sandreim, this will not works since the `custom-js` script will be ran in the pod that run zombienet and not in the `nodes`. I can add the feature to run `commands` in the `pods` through `kubectl/podman` as a new feature to validate this.

Thanks!

Reply:
Thanks for pointing that out. I made this assumption after so much local testing where this made sense. I think it would be helpful to have the ability to run commands inside the pod. Makes testing rely on more than just metrics and logs. We could also run a CPU profiler for example using this same interface.

Reply:
https://github.com/paritytech/zombienet/issues/246